src/core/public/chrome/ui/header/collapsible_nav_group_enabled.scss

```diff
@@ -114,4 +114,25 @@
     padding: 0 $euiSizeS;
     padding-left: $euiSize;
   }
+
+  .searchBar-wrapper {
+    padding: $euiSize;
+    padding-top: $euiSizeS;
+    background-color: transparent;
+    flex-grow: 0;
+
+    .searchInput {
+      background-color: transparentize(white, 0.5);
```

Comment:
Should we use some variable from theme? Will `white` work in dark mode?

Reply:
good catch, dark mode looks not correct with white, updated to variable `$ouiFormBackgroundColor`